NEMO/utilities.py

```diff
@@ -74,13 +92,43 @@ def convert_py_format_to_js(string_format: str) -> str:
     return string_format
 
 
+def convert_py_to_pick_time(string_format: str) -> str:
+    try:
+        return settings.PICK_TIME_FORMAT
```

Comment:
the setting name probably needs to be passed as argument, then use `getattr` so you can get the correct one (time or date)

Reply:
found a similar approach which actually helps to simplify :)

Reply:
that's perfect!